### PR TITLE
Added Multiple Linked List algorithm.

### PR DESCRIPTION
**Pull request by swapnanildutta**
Added the multiple linked list algorithm which can create more than one linked list at a time.

### CODE REVIEW

File: data_structures/linked_list/multiple_linked_list.c [hunk 1]
```diff
@@ -0,0 +1,106 @@
+#include<stdio.h>
```
Comment (deadshotsb) [inline data_structures/linked_list/multiple_linked_list.c:1]: This repository is purely for educational purposes therefore kindly add a description of your code and some sample test cases for better understanding of users, use comment lines.

Comment (kvedala) [reply data_structures/linked_list/multiple_linked_list.c:1]: 👍 Moreover, this is a good C program and not a C++ program and hence should be pulled to https://github.com/TheAlgorithms/C

Comment (swapnanildutta) [reply data_structures/linked_list/multiple_linked_list.c:1]: @kvedala I have pulled to https://github.com/TheAlgorithms/C itself.

Comment (kvedala) [reply data_structures/linked_list/multiple_linked_list.c:1]: > @kvedala I have pulled to https://github.com/TheAlgorithms/C itself.

Oh yes, my bad 😄

Comment (kvedala) [reply data_structures/linked_list/multiple_linked_list.c:1]: > This repository is purely for educational purposes therefore kindly add a description of your code and some sample test cases for better understanding of users, use comment lines.

Please add documentation following doxygen format like [this](https://github.com/kvedala/C/blob/master/numerical_methods/durand_kerner_roots.c). This enables the automation tools to better generate the documentation like [this](https://kvedala.github.io/C/da/d38/durand__kerner__roots_8c.html)
Thank you

**Comment by kvedala** [comment]
@swapnanildutta any update on this?

**Pull request closed without merging**